modules/images-configuration-shortname.adoc

```diff
@@ -17,9 +17,18 @@ When pulling or pushing images, the container runtime searches the registries li
 
 [WARNING]
 ====
-Using image short names with public registries is strongly discouraged. You should use image short names with only internal or private registries.
+Using image short names with public registries is strongly discouraged, because the image might not deploy if the public registry requires authentication. Use fully-qualified image names with public registries.
```

Comment:
```suggestion
Using image short names with public registries is strongly discouraged because the image might not deploy if the public registry requires authentication. Use fully-qualified image names with public registries.
```

Reply:
Thanks, Mike. I applied this update.